### ISSUE TITLE
Does Bigtable have concept of URI or anything similar?

### ISSUE THREAD

**Issue by matthewmturner** [datafusion-contrib/datafusion-bigtable#2]
I'm trying to think if this could be accessed using just SQL. I was thinking something like 
`CREATE EXTERNAL TABLE abc STORED AS BIGTABLE LOCATION uri`

**Comment by jychen7** [comment]
yes (or no), bigtable have uri like `https://googleapis.com/bigtable/projects/project_id/instances/instance_id/tables/table_name` ([source](https://cloud.google.com/bigquery/external-data-bigtable))

it does not have friendly name/schema like `gs://`, however, we can support custom format in Datafusion, e.g. `bigtable://{project_id}/{instance_id}/{table_name}`

**Comment by matthewmturner** [comment]
Ok, thx. Im focusing on cfg driven approach for now.  Will revist uri in the future if needed.  I will close this unless its something you are interested in.

**Comment by jychen7** [comment]
I think about adding to `datafusion` DDL, but it seems requiring `datafusion` requires `datafusion-bigtable` as optional dependency similar to https://github.com/apache/arrow-datafusion/pull/2111?

**Comment by matthewmturner** [comment]
I was hoping to eventually enable custom SQL in which case the relevant SQL keywords could be bundled with packages such as this rather than in datafusion itself.

**Comment by jychen7** [comment]
yes, agree

**Comment by jychen7** [comment]
> I will close this unless its something you are interested in.

@matthewmturner  thanks, I agree we can close it first
